README.md

```
@@ -92,6 +92,8 @@ Recursive DI
 
 - [x] 사전에 주어진 테스트 코드가 모두 성공해야 한다.
 
+--- 
```

markdownlint-cli2 (0.18.1)

Horizontal rule style
Expected: ---; Actual: --- 

(MD035, hr-style)